File: add/codesnippet/CSharp/p-system.windows.forms.i_30_1.cs
 public void MyLayoutName() {
    // Gets the current input language.
    InputLanguage myCurrentLanguage = InputLanguage.CurrentInputLanguage;
 
    if(myCurrentLanguage != null) 
       textBox1.Text = "Layout: " + myCurrentLanguage.LayoutName;
    else
       textBox1.Text = "There is no current language";
 }

File: add/codesnippet/CPP/p-system.windows.forms.i_30_1.cpp
public:
   void MyLayoutName()
   {
      // Gets the current input language.
      InputLanguage^ myCurrentLanguage = InputLanguage::CurrentInputLanguage;

      if ( myCurrentLanguage != nullptr )
      {
         textBox1->Text = String::Format( "Layout: {0}", myCurrentLanguage->LayoutName );
      }
      else
      {
         textBox1->Text = "There is no current language";
      }
   }